numpy/core/src/private/mem_overlap.c

```diff
@@ -535,15 +536,25 @@ solve_diophantine(unsigned int n, diophantine_term_t *E, npy_int64 b,
         return MEM_OVERLAP_NO;
     }
     else {
-        diophantine_term_t Ep[n];
-        npy_int64 Epsilon[n], Gamma[n];
+
+	
+	
+        diophantine_term_t *Ep = malloc(n * sizeof(diophantine_term_t));
+        npy_int64 *Epsilon = malloc(n * sizeof(npy_int64));
+		npy_int64 *Gamma = malloc(n *sizeof(npy_int64));
         Py_ssize_t count = 0;
 
         if (diophantine_precompute(n, E, Ep, Gamma, Epsilon)) {
+			free(Ep);
```

Comment:
Indentation looks off, I suspect hard tabs. Numpy C style requires tabs be 4 spaces.